ui/lib/replication/addon/templates/mode/secondaries/add.hbs

```diff
@@ -6,72 +6,74 @@
+      Secondary ID
+    </label>
+    <div class="control">
+      {{input class="input" name="activation-token-id" id="activation-token-id" value=id data-test-replication-secondary-id=true}}
     </div>
-  {{else}}
-    <div class="field">
-      <label for="activation-token-id" class="is-label">
-        Secondary ID
-      </label>
-      <div class="control">
-        {{input class="input" name="activation-token-id" id="activation-token-id" value=id data-test-replication-secondary-id=true}}
-      </div>
-      <p class="help has-text-grey">
-        This will be used to identify secondary cluster once a connection has been established with the primary.
-      </p>
-    </div>
-    <div class="field">
-      {{ttl-picker onChange=(action (mut ttl)) class="is-marginless"}}
-      <p class="help has-text-grey">
-        This is the Time To Live for the generated secondary token. After this period, the generated token will no longer be valid.
-      </p>
+    <p class="help has-text-grey">
+      This will be used to identify secondary cluster once a connection has been established with the primary.
+    </p>
+  </div>
+  <div class="field">
+    {{ttl-picker onChange=(action (mut ttl)) class="is-marginless"}}
+    <p class="help has-text-grey">
+      This is the Time To Live for the generated secondary token. After this period, the generated token will no longer be valid.
+    </p>
+  </div>
+  {{#if (eq replicationMode "performance")}}
+    <PathFilterConfigList
+      @paths={{paths}}
+      @config={{filterConfig}}
+      @id={{id}}
+    />
+  {{/if}}
+  <div class="field is-grouped box is-fullwidth is-bottomless">
+    <div class="control">
+      <button
+        type="submit"
+        class="button is-primary"
+        data-test-secondary-add=true
+        >
+        Generate token
+      </button>
     </div>
-    {{#if (eq replicationMode "performance")}}
-      <PathFilterConfigList
-        @paths={{paths}}
-        @config={{filterConfig}}
-        @id={{id}}
-      />
-    {{/if}}
-    <div class="field is-grouped box is-fullwidth is-bottomless">
-      <div class="control">
-        <button
-          type="submit"
-          class="button is-primary"
-          data-test-secondary-add=true
-          >
-          Generate token
-        </button>
-      </div>
-      <div class="control">
-        {{#link-to "mode.secondaries" replicationMode class="button"}}
-          Cancel
-        {{/link-to}}
-      </div>
+    <div class="control">
+      {{#link-to "mode.secondaries" replicationMode class="button"}}
+        Cancel
+      {{/link-to}}
     </div>
-  {{/if}}
+  </div>
 </form>
+
+{{#if isModalActive}}
+  <Modal @title="Copy your token" @onClose={{action "toggleModal" "Token copied!"}} @isActive={{isModalActive}}>
+    <section class="modal-card-body">
+      <p>This token can be used to enable {{replicationMode}} replication or change primaries on the secondary cluster.</p>
```

Comment:
looks like this needs to be updated to `Disaster Recovery Replication` instead of`dr replication`:

![image](https://user-images.githubusercontent.com/903288/83816510-69871180-a677-11ea-8a6f-45e744f392bf.png)

Reply:
We have this issue elsewhere in the app.  Therefore I went ahead and added a cluster property called `replicationModeForDisplay` and replaced elsewhere in the app where we hardcode or conditionally render the display version of the mode.  I made this one commit as Chelsea will likely need to implement these changes into her manage actions sidebranch.